scalableminds/webknossos
# Increase maximum bucket count limit for segmentation layers

This is a bandaid fix to lower the chance that annotating large amounts of data can cause a `A bucket was forcefully garbage-collected. This indicates that too many buckets are currently in RAM.` message. Especially, when annotating large structures in mag 1 (e.g., somata) a high amount of buckets can be labeled very quickly. Currently, wK does not cope very well with this. Increasing the threshold should help a bit (temporarily).

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- I tested that the session start time is sent to airbrake correctly.
- I opened a dataset with a segmentation layer to check that it loads+renders correctly

### Issues:
- related to #5973

------
(Please delete unneeded items, merge only when none are left open)
